tests/fixtures/orsay_museum.json

```diff
@@ -277,6 +277,26 @@
       "key": "phone",
       "value": "+33140494814"
     },
+    {
+        "key": "contact:facebook",
+        "value": "MuseeOrsay"
+    },
+    {
+        "key": "contact:twitter",
+        "value": "@MuseeOrsay"
+    },
+    {
+        "key": "contact:instagram",
+        "value": "https://www.instagram.com/MuseeOrsay"
+    },
+    {
```

Comment:
This key is duplicated. Is it on purpose?

Reply:
Nice catch :facepalm: